Modify wada uchu forum

yuichitakamizu opened the pull request:
please delete old two pdf files in imga

aywander commented:
Thanks! I fixed some stuff in the headers and changed his photo to the one from his webpage.

See: 9f26c26e41d960ac19f1effdb4c8a3a42a46bf3d

Just to note `affil` is the affiliation of the speaker, so in Wada-san's case its  University of Kagoshima.

yuichitakamizu commented:
Alex

Thanks! I noticed the face photo looks as just  imag. It's bag??
Yuichi

2018-05-11 16:25 GMT+09:00 Alex Y. Wagner <notifications@github.com>:

> Thanks! I fixed some stuff in the headers and changed his photo to the one
> from his webpage.
>
> See: 9f26c26
> <https://github.com/TheoreticalAstrophysicsGroup/TheoreticalAstrophysicsGroup.github.io/commit/9f26c26e41d960ac19f1effdb4c8a3a42a46bf3d>
>
> Just to note affil is the affiliation of the speaker, so in Wada-san's
> case its University of Kagoshima.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheoreticalAstrophysicsGroup/TheoreticalAstrophysicsGroup.github.io/pull/101#issuecomment-388283741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlO-O_rPvZo0ZJuRGmK7iin8qtg8iL6Eks5txTzsgaJpZM4T6-2Q>
> .
>

yuichitakamizu commented:
FINE! Yuichi

2018-05-11 16:42 GMT+09:00 yuichi takamizu <ytakamizu@gmail.com>:

> Alex
>
> Thanks! I noticed the face photo looks as just  imag. It's bag??
> Yuichi
>
> 2018-05-11 16:25 GMT+09:00 Alex Y. Wagner <notifications@github.com>:
>
>> Thanks! I fixed some stuff in the headers and changed his photo to the
>> one from his webpage.
>>
>> See: 9f26c26
>> <https://github.com/TheoreticalAstrophysicsGroup/TheoreticalAstrophysicsGroup.github.io/commit/9f26c26e41d960ac19f1effdb4c8a3a42a46bf3d>
>>
>> Just to note affil is the affiliation of the speaker, so in Wada-san's
>> case its University of Kagoshima.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/TheoreticalAstrophysicsGroup/TheoreticalAstrophysicsGroup.github.io/pull/101#issuecomment-388283741>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AlO-O_rPvZo0ZJuRGmK7iin8qtg8iL6Eks5txTzsgaJpZM4T6-2Q>
>> .
>>
>
>
